src/scripts/components/stories/story-media/story-media.tsx

```diff
@@ -6,19 +6,24 @@ import {NextIcon} from '../../main/icons/next-icon';
 import {FullscreenExitIcon} from '../../main/icons/fullscreen-exit-icon';
 import {FullscreenIcon} from '../../main/icons/fullscreen-icon';
 import {getStoryMediaUrl} from '../../../libs/get-story-media-url';
+import {useInterval} from '../../../hooks/use-interval';
+
+import {StoryMode} from '../../../types/story-mode';
 
 import styles from './story-media.styl';
 
 interface Props {
   images: string[];
   imageCaptions?: string[];
   storyId: string;
+  mode: StoryMode | null;
 }
 
 const StoryMedia: FunctionComponent<Props> = ({
```

Comment:
Another thing but not related to this PR. It's a bit confusing that this component is called StoryMedia and not sth like StoryGallery or StoryImages. At first I thought StoryMedia is a wrapper component for images and videos etc.

Reply:
Good point, will rename it in a different PR. 👍